File: 1876-substrings-of-size-three-with-distinct-characters/1876-substrings-of-size-three-with-distinct-characters.js
/**
 * @param {string} s
 * @return {number}
 */
var countGoodSubstrings = function(s) {
    let res = 0
    for (let i=1; i < s.length-1; i++) {
        if (s[i-1] !== s[i] && s[i] !== s[i+1] && s[i-1] !== s[i+1]) res++
    }
    return res
};

File: 1876-substrings-of-size-three-with-distinct-characters/1876-substrings-of-size-three-with-distinct-characters.py
class Solution:
    def countGoodSubstrings(self, s: str) -> int:
        res = 0
        for i in range(0, len(s)-2):
            if s[i] != s[i+1] and s[i+1] != s[i+2] and s[i] != s[i+2]:
                res += 1
        return res